libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -1654,7 +1699,7 @@ public void onClick(DialogInterface dialog, int id) {
                 }
                 break;
             default:
-                if (mediaType.equals(MediaType.VIDEO)) {
+                if (mediaType == MediaType.VIDEO) {
```

Comment:
let's add `if (mediaType == MediaType.VIDEO || mediaType == MediaType.AUDIO) {` here and let the default intent handler do its job. This will cross check your number 3 point in the PR description:

~3. Click on audio file won't trigger play~

😄